src/core/js/device.js

```diff
@@ -61,7 +62,28 @@ define(function(require) {
 	    Adapt.trigger('device:resize', Adapt.device.screenWidth);
 
     }, 100);
-
+    var onOrientationChange = _.debounce(function() {
+        var windowOrientation = window.orientation,
+            windowOrientationName;
+        Adapt.device.orientation = windowOrientation;
+        switch (windowOrientation) {
+            case -90:
+            case 90:
+                $('html').removeClass("orientation-portrait");
+                windowOrientationName = 'landscape';
+                break;
+            default:
+                $('html').removeClass("orientation-landscape");
+                windowOrientationName = 'portrait';
+                break;
+        }
+        Adapt.device.orientationName = windowOrientationName;
+        $('html').addClass("orientation-" + windowOrientationName);
+        Adapt.trigger('device:orientationchange', windowOrientation, windowOrientationName);
```

Comment:
Not sure if we should have orientationchange as a single event. Orientation is stored in the public Adpat.device object, so it can be accessed in Eventhandlers bound to `device:resize` or `device:changed`.